src/skills-builder/skills-builder-header/SkillsBuilderHeader.jsx

```diff
@@ -1,25 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { useIntl } from '@edx/frontend-platform/i18n';
 import edXLogo from '../images/edX-logo.svg';
 import messages from './messages';
 
-const SkillsBuilderHeader = () => {
+const SkillsBuilderHeader = ({ isMedium }) => {
   const { formatMessage } = useIntl();
 
+  if (isMedium) {
+    return (
+      <div className="d-flex">
+        <h3 className="mb-0 text-white">
+          {formatMessage(messages.skillsBuilderHeaderTitleIsMedium)}
+        </h3>
```

Comment:
Now that I'm taking a second look at this, I'm worried if we have a potential a11y issue here.

In the previous version, the title was displayed inside an `H1`. In this updated version, we have an `H3` here. Should this be an `H1` _styled like_ an `H3` (adding the `h3` declaration to the `className`)?

If the title of the header should be an H1, shouldn't the mobile-sized version also be an H1? 

More: I was taught that we should only have a single H1 on the DOM. This looks like we may potentially have NO H1 if we default to an H3?